compiler/hash-alloc/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "hash-alloc"
+version = "0.1.0"
+authors = ["The Hash Language authors"]
+edition = "2018"
+
+[dependencies]
+rayon = "1.5"
+dashmap = "4.0"
+parking_lot = "0.11"
```

Comment:
What's this dependency for?

Reply:
Removed extraneous dependencies in the latest commit.